docs/config.rst

```diff
@@ -155,6 +155,16 @@ These are the available audio configurations. For specific use cases, see
     ``gst-inspect-1.0`` to see what output properties can be set on the sink.
     For example: ``gst-inspect-1.0 shout2send``
 
+.. confval:: audio/buffer_time
+
+    Buffer size in milliseconds.
+
+    Expects an integer above 0.
+
+    Sets the buffer size of the GStreamer queue. If you experience buffering
+    before track changes, it may help to increase this. The default is letting
```

Comment:
Yeah, you might need all of them. But I would rather keep this simple and expose just this simplified concept. We can always guesstimate buffers and bytes per ms if we find out we need to.